src/script/calling/CallingRepository.ts

```diff
@@ -1112,7 +1124,7 @@ export class CallingRepository {
       const parsedPayload = JSON.parse(payload);
       const messageType = parsedPayload.type as CALL_MESSAGE_TYPE;
       if (messageType === CALL_MESSAGE_TYPE.REJECT) {
-        return;
+        return void this.messageRepository.sendSelfCallingMessage(payload, conversation.qualifiedId);
```

Comment:
Haven't seen this return void statement. Could you explain what's happening here?

Reply:
It's the javascript `void` operator here https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/void

It's to be absolutely sure we don't return the value returned by `sendSelfCallingMessage`. 

Another way to do it would be
```js
this.messageRepository.sendSelfCallingMessage(payload, conversation.qualifiedId);
return;
```

In this case I thing `void` is pretty nice :)

Reply:
got it, I am familiar with the second syntax.